src/main/java/com/github/retro_game/retro_game/entity/UserPasswordResetTokenKey.java

```diff
@@ -0,0 +1,42 @@
+package com.github.retro_game.retro_game.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Embeddable
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserPasswordResetTokenKey implements Serializable {
+
+  @JoinColumn(name = "user_id", nullable = false, unique = true, updatable = false)
+  @OneToOne(fetch = FetchType.LAZY)
+  private User user;
+
+  public User getUser() {
+    return user;
+  }
+
+  public void setUser(User user) {
+    this.user = user;
+  }
```

Comment:
Same here, this could be replaced by `@Getter` and `@Setter` on the `user` field.